apps/fhir/server/authentication.py

```diff
@@ -150,7 +146,7 @@ def match_fhir_id(mbi_hash, hicn_hash, request=None):
     # Perform primary lookup using MBI_HASH
     if mbi_hash:
```

Comment:
Like here, we should probably check for mbi instead of mbi_hash, since we only care about the mbi now.

Reply:
fixed